sklearn/cluster/hierarchical.py

```diff
@@ -711,8 +711,19 @@ class AgglomerativeClustering(BaseEstimator, ClusterMixin):
             ``pooling_func`` has been deprecated in 0.20 and will be removed
             in 0.22.
 
+    distance_threshold : float, optional (default=None)
+        The distance threshold to cluster at. If not ``None``, ``n_clusters``
```

Comment:
I still think this needs a description. It isn't clear what this parameter does right now.